internal/pkg/skuba/kubernetes/versions_test.go

```diff
@@ -368,3 +368,20 @@ func TestMajorMinorVersion(t *testing.T) {
 		})
 	}
 }
+
+// Compiler ensures we don't remove some code that's still deprecating.
+// This ensure code _removal_ when ready to do so.
+func TestEnsureHyperKubeIsRemovedForSupportedVersions(t *testing.T) {
+	for kubernetesVersionStr := range supportedVersions {
+		kubernetesVersion := version.MustParseSemantic(kubernetesVersionStr)
+		version_compared_to_118, err := kubernetesVersion.Compare("1.18.0")
```

Comment:
nit: Golang preferred variable naming is camelcase.

Reply:
I will update this.